SwSpotify/spotify.py

```diff
@@ -112,13 +114,31 @@ def get_info_mac():
     return a[3], a[1]
 
 
+def get_info_web():
+    """
+    Uses chrome extension to get the data
+    """
+    sys.stderr = open(os.devnull, "w")
```

Comment:
You redirect stderr to devnull but I don't think you restore it?

Reply:
would this restore it:
```python
sys.stderr = sys.__stderr__
```

Reply:
I believe this is in order so I will mark as resolved for tm. Unresolv is it is still a problem.

Reply:
I think it should too, btw why do you want to suppress errors here?

Reply:
Because the base library which I am using `http.server` prints all requests to the server to stderror. I could not find an easy way to suppress this effect by overriding the class. If you have a better idea mention it as this is properly not the best way to do this. You should be able to replicate the printing to stderror by just executing the `spotify_web.py`.

Reply:
Right, makes sense. I haven't done this myself so I'm not sure but you could google or stackoverflow once I guess, or even ask around in the SO chat :)

Reply:
good suggestion. I found a solution where I can override the log function in the `HTTPServer` class. In accordance with this I have removed the reroute of stderror from `spotify.py`.